components/nexus-httpclient/src/main/java/org/sonatype/nexus/httpclient/config/AuthenticationConfiguration.java

```diff
@@ -33,4 +35,13 @@ public String getType() {
   }
 
   // TODO: preemptive?
```

Comment:
depends of if we want to add a flag to enable preemptive auth or not.  I assume we did but wasn't sure how that was setup presently in the codebase.